### ISSUE TITLE
Update to remap-istanbul v0.11.1 to support new option 'warnMissingSourceMaps'

### ISSUE THREAD

**Issue by Nel75** [monounity/karma-typescript#283]
Please update to the latest remap-istanbul (i.e., 0.11.1 or later) so that the new option **warnMissingSourceMaps** is supported, to avoid bogus errors in the console about missing source maps for JS files in a hybrid application containing both TS and non-generated JS files.

Errors such as the following:
```
Error: Could not find source map for: "c:\...\someFile.js"
  at CoverageTransformer.addFileCoverage (c:\...\node_modules\karma-typescript\node_modules\remap-istanbul\lib\CoverageTransformer.js:171:18)

```

**Comment by erikbarke** [comment]
Absolutely, would you like to make a PR?

[Contributing guidelines](https://github.com/monounity/karma-typescript/wiki/Contributing)

**Comment by erikbarke** [comment]
The errors you're getting is because some of the example and integration projects can't be installed with a modern version of `npm`... Tracked in #294!

**Comment by erikbarke** [comment]
This should be fixed in #295 now, if you checkout that branch, does it run on your end ? After cloning you should be able to run:

```bash
npm run setup
npm run ci
```

I have only tested this with `npm`  v6.4.1 and `node` v8.12.0 and v10.12.0 🤔

**Comment by erikbarke** [comment]
Upgrade to remap-istanbul@0.12.0 in #296, closing!
